spec/features/request_a_visit_spec.rb

```diff
@@ -3,9 +3,7 @@
 RSpec.feature 'Booking a visit', js: true do
   include ActiveJobHelper
 
-  before do
-    create(:prison, name: 'Reading Gaol')
-  end
+  let!(:prison) { create(:prison, name: 'Reading Gaol') }
```

Comment:
It will, but I wanted it to be explicit in the feature. (I don't think it's a good default, and I think we'll have to change it eventually.)